docker/auto-setup.sh

```diff
@@ -61,7 +61,7 @@ set -eu -o pipefail
 
 : "${SKIP_DEFAULT_NAMESPACE_CREATION:=false}"
 : "${DEFAULT_NAMESPACE:=default}"
-: "${DEFAULT_NAMESPACE_RETENTION:=1d}"
```

Comment:
this was supported in tctl and I think old-new cli.. did we drop it in the new-new cli? we should fix that, everyone specifies retention in days

Reply:
We did, and you should reach out to our CLI team about that. I don't have the bandwidth to fix that right now